src/index.html

```diff
@@ -11,12 +11,95 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="stylesheet"
+      href="https://fonts.google.com/specimen/Roboto"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="main-header">
+      <a
+        href="#logo"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo company logo, showing the company name in blue and white"
+        />
+      </a>
+      <nav>
+        <ul class="nav-menu">
+          <li>
+            <a
+              href="#apple"
+              class="nav-link is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li>
+            <a
+              href="#samsung"
+              class="nav-link"
+            >
+              samsung
+            </a>
+          </li>
+          <li>
+            <a
+              href="#smartphones"
+              class="nav-link"
+            >
+              smartphones
+            </a>
+          </li>
+          <li>
+            <a
```

Comment:
Don't use spaces in `<a>` tag's `href` property. The `href` value `#laptops&computers` contains an unencoded space character.